packages/terra-table/src/Table.jsx

```diff
@@ -398,22 +400,26 @@ function Table(props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [tableColumns]);
 
+  const handleResize = () => {
+    setTableHeight(tableRef.current.offsetHeight - 1);
+
+    const tableContainer = tableContainerRef.current;
+    setTableScrollable(tableContainer.scrollWidth > tableContainer.clientWidth
+                      || tableContainer.scrollHeight > tableContainer.clientHeight);
+  };
+
+  const debouncedHandleResize = LodashDebounce(handleResize, 300);
```

Comment:
what is this 300 hardcoded value ?

Reply:
It is a 300ms delay before invoking the function.